### DISCUSSION TITLE
empty space

### DISCUSSION THREAD

**Discussion by Criaga** [Clooos/Bubble-Card#861] [Questions about config, custom styles and templates]
Good evening, I am using Bubble Card - Popup.

From the app, I have a blank part when I scroll down, how can I fix it?
 Thanks

https://github.com/user-attachments/assets/bbdc9e4c-bb7f-4329-bad9-95d68b2c1808

**Comment by MrBearPresident** [comment]
Did you solve this?
Can this be closed?
